generated/alertsmanagement/log.out

```diff
@@ -83,7 +83,12 @@ warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponse'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponse' -> 'ErrorResponseAutoGenerated2'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponseBody' -> 'ErrorResponseBodyAutoGenerated'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponseBody' -> 'ErrorResponseBodyAutoGenerated2'
-info    | autorest.bicep took 3ms
+warning | Skipping path '/subscriptions/{subscriptionid}/providers/microsoft.alertsmanagement/alerts/{alertid}/history': Found mismatch between type segments (2) and name segments (1)
```

Comment:
If this is applying to collection GETs, should it raise a warning? I added the warning to detect invalid PUT declarations (since a collection PUT is forbidden in the ARM RFC).

Reply:
The response from this get has the `x-ms-azure-resource` extension, but the [shape does look like a collection]( https://github.com/Azure/azure-rest-api-specs/blob/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2019-03-01/AlertsManagement.json#L1068). I think the behavior here is correct, but the Swagger is not.